redirects.yaml

```diff
@@ -47,6 +47,16 @@ build-snaps/build-for-another-arch/?: /t/building-the-snap/6800
 build-snaps/publish/?: /t/releasing-to-the-snap-store/6848
 build-snaps/register-snap/?: /t/registering-your-app-name/6793
 build-snaps/upload/?: /t/releasing-your-app/6795
+build-snaps/c/?: "/t/c-c-applications/7817"
```

Comment:
Some of those are already defined above in lines :27 - :37 - do we need these duplicates?